channels/client/addin.c

```diff
@@ -130,8 +129,8 @@ FREERDP_ADDIN** freerdp_channels_list_client_static_addins(LPSTR pszName, LPSTR
 				goto error_out;
 			}
 
-			strcpy(pAddin->cName, CLIENT_STATIC_ADDIN_TABLE[i].name);
-			strcpy(pAddin->cSubsystem, subsystems[j].name);
+			sprintf_s(pAddin->cName, ARRAYSIZE(pAddin->cName), "%s", CLIENT_STATIC_ADDIN_TABLE[i].name);
+			_snprintf(pAddin->cSubsystem, ARRAYSIZE(pAddin->cSubsystem), "%s", subsystems[j].name);
```

Comment:
What about silently truncating the input string (CLIENT_STATIC_ADDIN_TABLE[i].name)?
Either the pAddin->cName has no reason to be of limited size, and then it should be allocated dynamically to allow for the size of the names from the table,
or there's a good reason to limit the size of cName (eg. protocol or standard specification), and then the fact that the names in the tables are too big should be signaled as an error (preferably a compilation-time error, since they're literal, but a run-time error will do too).
In any case, it seems to be rather bad to silently and blindly truncate data.
(But still better than strcpy in this case).